client/src/js/components/resource.js

```diff
@@ -245,7 +245,7 @@ export default class Resource extends Component {
                     {this.fields().map(field => {
                         if (field === 'title') {
                             return (
-                                <div className="cloudinary-field__field">
+                                <div className="cloudinary-field__field" key={ field }>
```

Comment:
The `key` will always have a value of `title` it seems

Reply:
Which is fine, as long as it's unique for that instance. There's only one title field for an instance of a Cloudinary field.